docs/varnish/vcl/varnish7.vcl

```diff
@@ -0,0 +1,335 @@
+// Varnish VCL for:
+// - Varnish 7.1 or higher
```

Comment:
It would be good if you in the first commit only copied varnsh5.vcl as-is. So that in the this PR I could acutally see changed in the varnish6 vs varnish7...
As it is not, I have to diff them locally in order to see the difference

Reply:
yes. but if you first added the file as-is (a carbon copy of varnsish5.vcl, I could see the changes per commit.. The PR overview would still show it as a new file, correct

Reply:
Rebased